vcpkg/consuming-packages/configure-git-authentication-in-github-actions.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Configure Git authentication in GitHub Actions
+description: Learn to configure Git credentials for private repositories when using vcpkg on GitHub Actions
+author: vicroms
+ms.author: viromer
+ms.date: 7/20/2023
+ms.prod: vcpkg
+---
+# Configure Git authentication in GitHub Actions
+
+When using vcpkg with private registries it is necessary to provide access
+credentials to authenticate Git operations executed internally by vcpkg.
+
+In this article we will show you how to authenticate to a private vcpkg registry
+hosted on GitHub to use in your GitHub Actions workflows.
```

Comment:
What happens if you do this for a publicly visible vcpkg registry? Anything bad?

Reply:
Nothing bad happens, but authenticating to public repositories is unnecessary.